src/core/examples.js

```diff
@@ -61,16 +61,20 @@ export function run(conf) {
 
       // wrap
       example.classList.remove("example", "illegal-example");
+      // relocate the id to the div
+      const id = example.id ? example.id : null;
+      if (id !== "") {
```

Comment:
nit: assigning `null` but comparing to `""`

Reply:
oh, good catch!